### PR TITLE
Prototype

### PR DESCRIPTION
**Pull request by bryanhanner**
I set UDP_PORT to 8888. I did not see anything else I needed to do to initialize the Arduino for UDP communication. Please let me know if there is anything else I need to add.

Thank you,

Bryan

### CODE REVIEW

File: prototypeCode/roverCode/prototype/wirelessCommunication.ino [hunk 1]
```diff
@@ -7,7 +8,11 @@
...
 byte MAC_ADDRESS[] = {0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED};
-IPAddress IP(192, 168, 1, 4);
+int UPD_PORT = 8888;
+IPAddress IP(192, 168, 1, 51);
+char packetBuffer[UDP_TX_PACKET_MAX_SIZE];
+int x = 0;
+int y = 0;
 
 void initializeWirelessCommunication()
 {
```
Comment (twpryor) [inline prototypeCode/roverCode/prototype/wirelessCommunication.ino]: You need to put some commands in this function. Additionally you can't have all of your subsequent function declarations inside of this function. C++ doesn't allow declaring a function inside of another function.

File: prototypeCode/roverCode/prototype/wirelessCommunication.ino [hunk 2]
```diff
@@ -18,11 +23,56 @@ void initializeWirelessCommunication()
...
+            {
+                x = x*10 + (packetBuffer[i] - 48);
+            }
+            else
+            {
+                y = y*10 + (packetBuffer[i] - 48);
+            }
+        }
+    }
 }
```
Comment (twpryor) [inline prototypeCode/roverCode/prototype/wirelessCommunication.ino]: Move this last bracket up to close ``` initializeWirelessCommunication() ```

**Comment by twpryor** [comment]
Checkout this link to see what the setup commands are:
https://www.arduino.cc/en/Tutorial/UDPSendReceiveString

**Comment by bryanhanner** [comment]
In response to your 3 emails:

1. I believe I have commands in the function. Should I have other ones? I
also don't see any functions declared inside other functions.

2. I checked my curly brackets, and they all seem to line up. I am not sure
which bracket should be moved.

3. It seems like the commands in the setup method on the website are the
same as the ones in my initializeWirelessCommunication method, excluding
the Serial command since we are not using the Serial on the rover. Is there
something else I should have? On a different topic, I added some lines of
code to the getPacket method that I was missing to acknowledge that the
packet was received.

Bryan

**Comment by twpryor** [comment]
Git hub was folding portions of the code for me and it looked like you hadn't closed the function ```initializeWirelessCommunication()``` but once I viewed the whole file, everything showed up. My bad! The code looks good now.

**Comment by bryanhanner** [comment]
Good to hear! Thanks!

**Pull request merged**